analyzer/tests/unit/test_checker_handling.py

```diff
@@ -396,6 +399,20 @@ def test_disable_clangsa_checkers(self):
                 analyzer.config_handler.checks()['Wreserved-id-macro'][0],
                 CheckerState.enabled)
 
+    def test_analyze_wrong_parameters(self):
+        """
+        This test checks whether the analyze command detects if a wrong
+        --analyzer-config or --checker-config parameter is specified.
+        """
+
+        analyzer_cfg = [AnalyzerConfig('asd', '123', 'value')]
+        checker_cfg = [CheckerConfig('clangsa', 'asd', '123', 'value')]
+
+        analyzer_func = validate_analyzer_parameter(analyzer_cfg)
+        checker_func = validate_checker_parameter(checker_cfg)
+
+        [self.assertFalse(func) for func in [analyzer_func, checker_func]]
```

Comment:
This code looks smarter than it should be :^)
```suggestion
        self.assertFalse(validate_analyzer_parameter(analyzer_cfg))
        self.assertFalse(validate_checker_parameter(checker_cfg)
```